revocation/revocation.go

```diff
@@ -44,9 +44,9 @@ type ValidateContextOptions struct {
 	// been validated. REQUIRED.
 	CertChain []*x509.Certificate
 
-	// SigningTime denotes the signing time of the signature.
+	// AuthenticSigningTime denotes the authentic signing time of the signature.
 	// OPTIONAL.
```

Comment:
Do you want to add back 
> 	// It is solely used under signing scheme `notary.x509.signingAuthority`.

Reply:
This is comment from Pritesh: https://github.com/notaryproject/notation-core-go/pull/215#discussion_r1704635550

Reply:
Added more comments.